conf.py

```diff
@@ -32,14 +34,22 @@
     '.github',
 ]
 
+def _asset_hash(path: str) -> str:
```

Comment:
```suggestion
def _asset_hash(path: os.PathLike[str]) -> str:
```

This works with any `str` path-like object (e.g. `pathlib.Path`) too.